integration_tests/__init__.py

```diff
@@ -366,6 +366,12 @@ def setUp(self):
         self.test_store = fixture_setup.TestStore()
         self.useFixture(self.test_store)
 
+    def is_store_fake(self):
+        return (os.getenv('TEST_STORE') or 'fake') == 'fake'
```

Comment:
`os.getenv('TEST_STORE', 'fake')` would look nicer, I think.

Reply:
This was part of the bug. It would return '' if TEST_STORE is set to ''. Perhaps counter intuitively, getenv only uses the default if a variable is unset.

Reply:
Ah interesting, I suppose that makes sense. Ignore me, then.